Unable to obtain connection from database (jdbc:sqlite:/home/chris/.bitcoin-s/oracle/oracle.sqlite) for user '': opening db: '/home/chris/.bitcoin-s/oracle/oracle.sqlite':

When i try to start the oracle server on  1f63d24629571f88e81414cc8912d0a90025e20e I get this stack trace

This was probably a regression introduced in #2667 

```
2021-02-24T13:53:17UTC INFO [ActorSystemImpl] Akka started
2021-02-24T13:53:18UTC INFO [VersionPrinter] Flyway Community Edition 6.4.2 by Redgate
2021-02-24T13:53:18UTC WARN [DLCOracleAppConfig] Failed to apply first round of migrations, attempting baseline and re-apply
org.flywaydb.core.internal.exception.FlywaySqlException:
Unable to obtain connection from database (jdbc:sqlite:/home/chris/.bitcoin-s/oracle/oracle.sqlite) for user '': opening db: '/home/chris/.bitcoin-s/oracle/oracle.sqlite': Not a directory
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
SQL State  : null
Error Code : 0
Message    : opening db: '/home/chris/.bitcoin-s/oracle/oracle.sqlite': Not a directory

        at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:65)
        at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:80)
        at org.flywaydb.core.Flyway.execute(Flyway.java:456)
        at org.flywaydb.core.Flyway.migrate(Flyway.java:159)
        at org.bitcoins.db.DbManagement.migrate(DbManagement.scala:160)
        at org.bitcoins.db.DbManagement.migrate$(DbManagement.scala:157)
        at org.bitcoins.dlc.oracle.config.DLCOracleAppConfig.migrate(DLCOracleAppConfig.scala:22)
        at org.bitcoins.dlc.oracle.config.DLCOracleAppConfig.$anonfun$start$2(DLCOracleAppConfig.scala:66)
        at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:434)
        at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:53)
        at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: java.sql.SQLException: opening db: '/home/chris/.bitcoin-s/oracle/oracle.sqlite': Not a directory
        at org.sqlite.SQLiteConnection.open(SQLiteConnection.java:226)
        at org.sqlite.SQLiteConnection.<init>(SQLiteConnection.java:61)
        at org.sqlite.jdbc3.JDBC3Connection.<init>(JDBC3Connection.java:28)
        at org.sqlite.jdbc4.JDBC4Connection.<init>(JDBC4Connection.java:21)
        at org.sqlite.JDBC.createConnection(JDBC.java:115)
        at org.sqlite.JDBC.connect(JDBC.java:90)
        at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:474)
        at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:440)
        at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:56)
        ... 15 common frames omitted
2021-02-24T13:53:18UTC ERROR [OracleServerMain] Failed to startup server!
org.flywaydb.core.internal.exception.FlywaySqlException:
Unable to obtain connection from database (jdbc:sqlite:/home/chris/.bitcoin-s/oracle/oracle.sqlite) for user '': opening db: '/home/chris/.bitcoin-s/oracle/oracle.sqlite': Not a directory
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
SQL State  : null
Error Code : 0
Message    : opening db: '/home/chris/.bitcoin-s/oracle/oracle.sqlite': Not a directory

        at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:65)
        at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:80)
        at org.flywaydb.core.Flyway.execute(Flyway.java:456)
        at org.flywaydb.core.Flyway.baseline(Flyway.java:344)
        at org.bitcoins.db.DbManagement.migrate(DbManagement.scala:168)
        at org.bitcoins.db.DbManagement.migrate$(DbManagement.scala:157)
        at org.bitcoins.dlc.oracle.config.DLCOracleAppConfig.migrate(DLCOracleAppConfig.scala:22)
        at org.bitcoins.dlc.oracle.config.DLCOracleAppConfig.$anonfun$start$2(DLCOracleAppConfig.scala:66)
        at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:434)
        at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:53)
        at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: java.sql.SQLException: opening db: '/home/chris/.bitcoin-s/oracle/oracle.sqlite': Not a directory
        at org.sqlite.SQLiteConnection.open(SQLiteConnection.java:226)
        at org.sqlite.SQLiteConnection.<init>(SQLiteConnection.java:61)
        at org.sqlite.jdbc3.JDBC3Connection.<init>(JDBC3Connection.java:28)
        at org.sqlite.jdbc4.JDBC4Connection.<init>(JDBC4Connection.java:21)
        at org.sqlite.JDBC.createConnection(JDBC.java:115)
        at org.sqlite.JDBC.connect(JDBC.java:90)
        at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:474)
        at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:440)
        at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:56)
        ... 15 common frames omitted
```

--- a/dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/config/DLCOracleAppConfig.scala
+++ b/dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/config/DLCOracleAppConfig.scala
@@ -2,7 +2,7 @@ package org.bitcoins.dlc.oracle.config
 
 import com.typesafe.config.Config
 import org.bitcoins.core.api.dlcoracle.db.EventOutcomeDbHelper
-import org.bitcoins.core.config.NetworkParameters
+import org.bitcoins.core.config._
 import org.bitcoins.core.crypto.ExtKeyVersion.SegWitMainNetPriv
 import org.bitcoins.core.hd.HDPurpose
 import org.bitcoins.core.protocol.tlv.EnumEventDescriptorV0TLV
@@ -61,8 +61,18 @@ case class DLCOracleAppConfig(
         Files.createDirectories(datadir)
       }
 
+      val networkDir = {
+        val lastDirname = network match {
+          case MainNet  => "mainnet"
+          case TestNet3 => "testnet3"
+          case RegTest  => "regtest"
+          case SigNet   => "signet"
+        }
+        baseDatadir.resolve(lastDirname)
+      }
+
       // Move old db in network folder to oracle folder
-      val oldNetworkLocation = datadir.resolve("oracle.sqlite")
+      val oldNetworkLocation = networkDir.resolve("oracle.sqlite")
       if (!exists() && Files.exists(oldNetworkLocation)) {
         Files.move(oldNetworkLocation, dbPath)
       }

--- a/dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/config/DLCOracleAppConfig.scala
+++ b/dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/config/DLCOracleAppConfig.scala
@@ -50,6 +50,10 @@ case class DLCOracleAppConfig(
   /** The path to our encrypted mnemonic seed */
   lazy val seedPath: Path = kmConf.seedPath
 
+  override lazy val datadir: Path = {
+    baseDatadir.resolve("oracle")
+  }
+
   override def start(): Future[Unit] = {
     logger.debug(s"Initializing dlc oracle setup")
     super.start().flatMap { _ =>

